MANIFEST.in

```diff
@@ -1,2 +1,3 @@
 include LICENSE.txt CHANGELOG.rst
 recursive-include demos *.py *.bat *.sh
+recursive-include tests *.py
```

Comment:
Thinking out loud. Do we need to include the tests in the package? I think https://hynek.me/articles/testing-packaging/ suggests we do not.

Could the test runner just import the tests from something like PROJECTROOT/tests ? (which will contain an __init__.py).

I might try it out myself tomorrow, but feel free to try it yourself if this makes sense to you.